var/spack/repos/builtin/packages/mivisionx/package.py

```diff
@@ -34,26 +35,34 @@ def url_for_version(self, version):
 
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
 
+    def patch(self):
+        if '@4.2.0' in self.spec:
+            filter_file('${ROCM_PATH}/opencl',
+                        self.spec['rocm-opencl'].prefix,
+                        'amd_openvx/cmake/FindOpenCL.cmake',
+                        string=True)
+
     def flag_handler(self, name, flags):
         spec = self.spec
         protobuf = spec['protobuf'].prefix.include
         if name == 'cxxflags':
             flags.append('-I{0}'.format(protobuf))
         return (flags, None, None)
 
+    depends_on('cmake@3.16.1', type='build')
```

Comment:
```suggestion
    depends_on('cmake@3.5:', type='build')
```

How do you get to this very particular version?

https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/search?p=1&q=cmake_minimum_required